flatten-openalex-jsonl.py

```diff
@@ -392,7 +392,7 @@ def flatten_institutions():
 
                     # associated_institutions
                     if associated_institutions := institution.get(
-                        'associated_institutions', institution.get('associated_insitutions')  # typo in api
+                        'associated_institutions', institution.get('associated_institutions')  # typo in api
```

Comment:
It first tries to access the correct key "associated_institutions" and then tries the misspelling "associated_insitutions" if not found/in case of API misspelling. This change would have have it attempt to access the same key twice and not account for misspelled case.